pages/splash.js

```diff
@@ -0,0 +1,74 @@
+import Head from "next/head";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { useTranslation } from "next-i18next";
+import { ActionButton } from "../components/atoms/ActionButton";
+import Link from "next/link";
+
+export default function splash(props) {
+  const { t } = useTranslation("common");
+
+  return (
+    <div>
+      <div className="bg-banner-img h-screen min-w-300px min-h-screen blur" />
+      <Head>
+        <title>{t("splashTitle")}</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <div className="absolute inset-0 m-auto flex flex-col justify-center items-center">
+        <div className="bg-footer-background-color h-230px min-w-300px w-300px xl:w-500px">
+          <img
+            className="h-auto w-64 container mx-auto pt-6 pb-8 xl:w-80 xl:mx-0 xl:px-6"
+            src={"/sig-blk-en.svg"}
+            alt={"Symbol of the Government of Canada"}
+          />
+          <div className="w-max container mx-auto pb-8 text-p font-medium">
+            alpha.service.canada.ca
+          </div>
+          <div className="flex w-max container mx-auto">
+            <ActionButton
```

Comment:
Buttons should be more rounded and the text smaller on the splash page 

Current button
![Screen Shot 2021-05-05 at 11 06 23 AM](https://user-images.githubusercontent.com/26819992/117163724-f065d200-ad91-11eb-8047-5d31e3e986e0.png)

Splash page button
![Screen Shot 2021-05-05 at 11 06 17 AM](https://user-images.githubusercontent.com/26819992/117163833-0a9fb000-ad92-11eb-87f5-e3251161a897.png)